presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcRecordSink.java

```diff
@@ -130,6 +132,11 @@ public void appendLong(long value)
                 long localMillis = ISOChronology.getInstanceUTC().getZone().getMillisKeepLocal(DateTimeZone.getDefault(), utcMillis);
                 statement.setDate(next(), new Date(localMillis));
             }
+            else if (TIMESTAMP.equals(columnTypes.get(field))) {
+                long utcMillis = TimeUnit.MILLISECONDS.toMillis(value);
```

Comment:
This is the same portion of code as below, we decide to keep it as this for now.